python_functionality/homework.py

```diff
@@ -25,8 +26,8 @@ def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
        remove_dict_field([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 'age')
         >>> [{'name': 'Alex'}, {'name': 'denys'}]
     """
-    pass
-
+    l = [{key: val for (key, val) in i.items() if key not in redundant_keys} for i in data]
+    return l
```

Comment:
you do not need to create a variable and just return her. Just return list comp
There is a rule: 'Create variable if you gonna use her more than 2 times'. Of course, this rule is not cover all cases, but most